solidity/test/DepositRedemptionTest.js

```diff
@@ -916,16 +916,17 @@ describe("DepositRedemption", async function() {
       )
     })
 
-    it("approves a new digest for signing, updates the state, and logs RedemptionRequested", async () => {
+    it("correctly increases fee, approves a new digest for signing, updates the state, and logs RedemptionRequested", async () => {
```

Comment:
Feels like there's also a state test here that does a couple of increase fee bump tests and then confirms that you can redeem with the original fee, an intermediate fee, the final fee that was set, and cannot redeem with a fee higher than the one that was set. Thoughts?